cl/audio/templates/oral_argument.html

```diff
@@ -203,7 +203,7 @@ <h3>{{ af.docket.court }}</h3>
                     </div>
                     <div class="jp-no-solution">
                         <span>Update Required</span>
-                        To play the media update your browser to a recent version or update your <a href="http://get.adobe.com/flashplayer/"
+                        To play the media update your browser to a recent version or update your <a href="https://get.adobe.com/flashplayer/"
```

Comment:
This is no longer accurate.  Flash has been sunset.  Thank goodness

Reply:
Yeah, probably not, but we can solve it another day, out of scope for this (but I walked myself into this quandary by expanding my scope to all http links, doh!)

Reply:
I mean ... removing flash is always in scope to me... but I guess its a personal choice

Reply:
We'll get there. It's not causing any trouble in the meantime. When we get there, we can look at larger solutions to our janky audio player.